api/proto/common/v2/common_v2.proto

```diff
@@ -7,22 +7,38 @@ option go_package = "github.com/Layr-Labs/eigenda/api/grpc/common/v2";
 
 // BlobHeader contains the information describing a blob and the way it is to be dispersed.
 message BlobHeader {
-  // The blob version. Blob versions are pushed onchain by EigenDA governance in an append only fashion and store the
-  // maximum number of operators, number of chunks, and coding rate for a blob. On blob verification, these values
-  // are checked against supplied or default security thresholds to validate the security assumptions of the
-  // blob's availability.
+  // The BlobParams version to use when encoding the blob into chunks to be dispersed to operators.
+  //
+  // BlobParams versions are pushed onchain to the EigenDAThresholdRegistry by EigenDA governance in an append only fashion
+  // and store the maximum number of operators, number of chunks, and coding rate for a blob.
+  //
+  // A user can choose any of the onchain defined VersionedBlobParams, and must make sure to choose SecurityThresholds in its CertVerifier contract
+  // that along with the chosen VersionedBlobParams satisfy the checkSecurityParams function: https://github.com/Layr-Labs/eigenda/blob/3e670ff3dbd3a0a3f63b51e40544f528ac923b78/contracts/src/periphery/cert/libraries/EigenDACertVerificationLib.sol#L188
+  // This function is called internally by the CertVerifier's checkDACert function.
+  //
+  // If a version that is not available on the ThresholdRegistry is chosen, the disperser will return an error.
+  //
+  // EigenDA maintained:
+  //   VersionedBlobParams definition: https://github.com/Layr-Labs/eigenda/blob/3e670ff3dbd3a0a3f63b51e40544f528ac923b78/contracts/src/core/libraries/v1/EigenDATypesV1.sol#L7
+  //   IEigenDAThresholdRegistry (stores the BlobParams): https://github.com/Layr-Labs/eigenda/blob/3e670ff3dbd3a0a3f63b51e40544f528ac923b78/contracts/src/core/interfaces/IEigenDAThresholdRegistry.sol
+  //   EigenDAServiceManager address (implements IEigenDAThresholdRegistry): https://docs.eigenda.xyz/networks/mainnet#contract-addresses
+  // Rollup maintained:
+  //   SecurityThresholds interface: https://github.com/Layr-Labs/eigenda/blob/3e670ff3dbd3a0a3f63b51e40544f528ac923b78/contracts/src/periphery/cert/interfaces/IEigenDACertVerifier.sol#L23
+  //   checkDACert interface: https://github.com/Layr-Labs/eigenda/blob/3e670ff3dbd3a0a3f63b51e40544f528ac923b78/contracts/src/periphery/cert/interfaces/IEigenDACertVerifierBase.sol#L8
   uint32 version = 1;
-  // quorum_numbers is the list of quorum numbers that the blob is part of.
-  // Each quorum will store the data, hence adding quorum numbers adds redundancy, making the blob more likely to be retrievable. Each quorum requires separate payment.
+  // quorum_numbers is the list of quorum numbers that the blob shall be dispersed to.
+  // Each quorum will store the data independently, meaning that additional quorum numbers increase redundancy, making the blob more likely to be retrievable.
```

Comment:
> Each quorum will store the data independently

Is this still true, given the recent chunk assignment changes by @mooselumph? I didn't fully grasp those changes, but I recall something about not duplicating storage when a given node is in multiple quorums.

Reply:
Ah. But you could in theory have single quorum's worth of replication if all quorums align. Which I guess is what we want thinking about it...
ok so this new update removes the useless replication (same operator storing same data 3 times) but still keeps the meaningful replication (quorums having different operators will cause an actual replication of data)